### PR TITLE
fixed getEntryParserRule with actions [eclipse/xtext-xtend#1018]

### PR DESCRIPTION
**Pull request by cdietrich**
fixed getEntryParserRule with actions [eclipse/xtext-xtend#1018]
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.tests/src/org/eclipse/xtext/testlanguages/ActionTestLanguage3.xtext [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
+ * Copyright (c) 2008 itemis AG (http://www.itemis.eu) and others.
```
Comment (szarnekow) [inline org.eclipse.xtext.tests/src/org/eclipse/xtext/testlanguages/ActionTestLanguage3.xtext]: 2020

File: org.eclipse.xtext.tests/src/org/eclipse/xtext/testlanguages/ActionTestLanguage3.xtext [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ *******************************************************************************/
+grammar org.eclipse.xtext.testlanguages.ActionTestLanguage3 with org.eclipse.xtext.common.Terminals
+
+generate actionLang3 "http://www.eclipse.org/2008/tmf/xtext/ActionLang3"
```
Comment (szarnekow) [inline org.eclipse.xtext.tests/src/org/eclipse/xtext/testlanguages/ActionTestLanguage3.xtext]: 2020

File: org.eclipse.xtext.tests/src/org/eclipse/xtext/testlanguages/ActionTestLanguage3.xtext [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
```
Comment (szarnekow) [inline org.eclipse.xtext.tests/src/org/eclipse/xtext/testlanguages/ActionTestLanguage3.xtext]: Superb. Thank's for adding a test!

**Comment by cdietrich** [comment]
yes. this also worked before. i added the test. can you have a look if i did the test correct?

**Pull request merged**